classes/test/testcase_helper_trait.php

```diff
@@ -16,19 +16,21 @@
 
 namespace mod_bigbluebuttonbn\test;
 
-use advanced_testcase;
 use context_module;
+use PHPUnit\Framework\SkippedTestError;
```

Comment:
Why are these necessary?

Reply:
That was in the PHP Docs comment, added by mistake.